Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocReferenceValue.css

```diff
@@ -69,6 +69,13 @@
   cursor: default;
 }
 
+.bocReferenceValue.readOnly span.body > span.icon
+{
+  align-self: start;
+  padding-top: calc(var(--remotion-themed-spacing) - var(--remotion-themed-border-width));
+  border-top: 1px solid transparent;
```

Comment:
this feels strange. first you subtract 1px, then you add it again. I didn't analyze the rest of the file for the underlying reason, you elaborate (screenshot or live)? And either way, we have the border-width var that should be used here, too, in case there is a good reason why we need both a padding-top and a border-top.

Reply:
Yeah this part is sad. Because we did not include the padding in the grid layout simply using `align-self: start` will put the icon above the text (since the text has padding but the button does not). For some reason some magic pixels (~2px) are missing when I apply the same padding as the text element and it does not line up. So even if we use the same padding we are still off...

I don't know what the best calculation is that we can put here to have the correct spacing. Adapting the CSS grid to include the spacing would be ideal but this takes more time.